SPECS/vte291/vte291.spec

```diff
@@ -21,6 +21,7 @@ Source0:        https://download.gnome.org/sources/vte/%{majorver}/vte-%{version
 # https://bugzilla.redhat.com/show_bug.cgi?id=1103380
 # https://gitlab.gnome.org/GNOME/vte/-/issues/226
 Patch100:       vte291-cntnr-precmd-preexec-scroll.patch
+Patch101:       CVE-2024-37535.patch
```

Comment:
Looks like this spec is using the old `%setup` macro with single `%patch` macros for each patch. While we're here, maybe let's switch to `%autopatch`.